src/docs/development/add-to-app/ios/add-flutter-screen.md

```diff
@@ -0,0 +1,487 @@
+---
+title: Adding a Flutter screen to an iOS app
+short-title: Add a Flutter screen
+description: Learn how to add a single Flutter screen to your existing iOS app.
+---
+
+This guide describes how to add a single Flutter screen to an existing iOS app.
+
+## Integration Steps
+
+ 1. [Create Flutter Module](#create-a-flutter-module) - This creates the Flutter project which will be hosted in the existing app.
+ 1. [Add Module Dependency in Existing App](#make-the-host-app-depend-on-the-flutter-module) - This sets up XCode to include your Flutter project into the build of your existing app.
+ 1. [Add FlutterViewController to Existing App](#write-code-to-use-flutterviewcontroller-from-your-host-app) - This steps introduces the code that will conjure the instance of Flutter into your existing app.
+ 1. (optional) [Attach Flutter Tool](#building-and-running-your-app) - This step allows you to attach to Flutter's debugger and enables you do use hot reload / restart.
+
+## Create a Flutter module
+
+Flutter projects created using `flutter create xxx` include very simple host
+apps for your Flutter/Dart code (a single-ViewController iOS host). You can
+modify these host apps to suit your needs and build from there.
+
+But if you're starting off with an *existing* host app for either platform,
+you'll likely want to include your Flutter project in that app as some form of
+library instead.
+
+This is what the Flutter module template provides. Executing
+`flutter create -t module xxx` produces a Flutter project containing a CocoaPods
+pod designed for consumption by your existing host app.
+
+Let's assume you have an existing iOS app at `some/path/MyApp`, and that you
+want your Flutter project as a sibling:
+
+```bash
+cd some/path/
+flutter create -t module my_flutter
+```
+
+This creates a `some/path/my_flutter/` Flutter module project with some Dart
+code to get you started and a `.ios/` hidden subfolder that wraps up the
```

Comment:
```
to get you started and => to get you started, and a `.ios/` hidden subfolder.
The ios folder wraps the module containing some CocoaPods and a helper
Ruby script.
```

The original sentence was quite a mouthful, so I'm trying to break it up a bit.

Reply:
done